breakdown-guides.rst

```diff
@@ -0,0 +1,402 @@
+
+Expected Cost Impact
+	Automatically generated value based on the following formula:
+
+	``Minimum Cost`` + ``Likely Cost``*4 + ``Maximum Cost``/6
+
+Expected Monetary Value
+	Automatically generated value based on the following formula:
+
+	(``Minimum Cost`` + ``Likely Cost``*4 + ``Maximum Cost``/6) * ((``Likelihood Score`` * 0.20) - 0.10)
+
+Financial Provision
+	This field is not used by Rubin Observatory.
+
+Number of Possible Occurrences
+	Define the number of potential occurrences this risk can be realized.
+
+
+.. _Breakdown-Risk-Residual-Impact:
+
+Risk residual impact
+--------------------
+
+.. Use :ref:`Response Plans <Breakdown-Plan>` when section is ready.
+
+After a risk is realized, Response Plans are used to address it.
+The ``Residual Cost Impact``, ``Residual Schedule Impact`` and ``Residual Likelihood`` analyze the realized risk impact after the plan is activated.
+The methods and definitions for categorization are the same as :ref:`Breakdown-Risk-Initial-Impact` and :ref:`risk scores <Breakdown-Risk-Score-Quantitative>`.
+
+.. figure:: /_static/Risk-Example-Plans-Actions-Residual-Risk.png
+    :name: Risk-Example-Plans-Actions-Residual-Risk
+
+    Residual Risk Impacts and related Response Plans and Related Actions sections using an example risk.
+
+Plan Type
+	The ``plan type`` is the strategic process of controlling the identified risks via response plans.
+	There are four types of processes, and their implementation depends on if the risk is a threat or opportunity.
+
+.. figure:: /_static/Response-Plan-Types.png
+    :name: Response-Plan-Types
+
+    Four processes and respective threat or opportunity response plan types.
+
+Response Types for Threats
+	Avoid
+		Changing your strategy or plans to avoid the risk.
+		This risk response strategy is about removing the threat by any means.
+		That can mean changing your management plan to avoid the risk because it’s detrimental to the project/program.
+
+	Transfer
+		Passing ownership and/or liability to a third party to resolve the risk, e.g., purchase fire insurance for an unfinished building.
+
+	Mitigate
+		Reducing the probability and/or impact of the risk below a threshold of acceptability.
+		Some risks cannot be avoided and need to take action to reduce the impact of the risk, e.g., work procedures and equipment designed to reduce workplace safety risks.
+
+	Accept
+		Recognizing residual risks and devising responses to control and monitor them.
+		This risk response strategy consists of identifying a risk and documenting all the risk management information about it, but not taking any action unless the risk is realized.
+
+Response Types for Opportunities
```

Comment:
```suggestion
Some risks may have response plans of more than one type. In this case, specify plan type of the costliest present. The plan types in order of increasing costliness are: Accept (cheapest), Transfer, Avoid, Mitigate (most expensive). 

Response Types for Opportunities
```

Reply:
Added to next commit. We should state what the order is for opportunities (added todo item).